webapp/courses/static/courses/terms.css

```diff
@@ -43,26 +43,53 @@ div.term-description:before {
     height: 0;
     position: absolute;
     width: 0;
-    top: -18px;
-    left: 0px;
-    border: 9px solid transparent;
-    border-bottom-color: #ccc;
+    top: -23px;
+    left: -1px;
+    border: 11px solid transparent;
+    border-bottom-color: #999;
```

Comment:
What was wrong with the previous values? This is not listed in the commit notes...